### ISSUE TITLE
Problem when requireing the ext via composer:

### ISSUE THREAD

**Issue by josefglatz** [TYPO3-Initiatives/adminpanel-extended#17]
Problem: The ext is installed within typo3conf/ext/introduction

Here some infos from packagist:

```
composer info -a psychomieze/adminpanel-extended
Found composer.json directory: /Volumes/CS/Projekte/TYPO3/TYPO3-Distribution/TYPO3-Distribution/app
name     : psychomieze/adminpanel-extended
descrip. : TYPO3 AdminPanel Extended
keywords :
versions : 0.0.2, 0.0.1, dev-develop, dev-feature/_int-object-logging
type     : typo3-cms-extension
license  : GNU General Public License v2.0 or later (GPL-2.0+) (OSI approved) https://spdx.org/licenses/GPL-2.0+.html#licenseText
source   : [git] https://github.com/TYPO3-Initiatives/adminpanel-extended.git 267d785331930f1d2cd090f0bb797f59995990bc
dist     : [zip] https://api.github.com/repos/TYPO3-Initiatives/adminpanel-extended/zipball/267d785331930f1d2cd090f0bb797f59995990bc 267d785331930f1d2cd090f0bb797f59995990bc
names    : psychomieze/adminpanel-extended, introduction

autoload
psr-4
Psychomieze\AdminpanelExtended\ => Classes/

requires
typo3/cms-adminpanel 9.5.*@dev

requires (dev)
roave/security-advisories dev-master

replaces
introduction *
```

My first thought: do you activated the GitHub hook for packagist?

**Comment by josefglatz** [comment]
It only works if I require it with `dev-develop`

**Comment by KamiYang** [comment]
Thanks for the hint. 
The problem exists because in the current released versions (0.0.1 & 0.0.2) the `replace` section is wrong. https://github.com/TYPO3-Initiatives/adminpanel-extended/blob/267d785331930f1d2cd090f0bb797f59995990bc/composer.json#L13-L15
This has been fixed in the development branch. For future releases this wont occur. Means we should release a new version ASAP.

EDIT:
Like you already mentioned, the workaround is using `composer req psychomieze/adminpanel-extended:dev-develop` for now.

**Comment by KamiYang** [comment]
Shouldn‘t appear anymore with the current release.
